src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/CurrentLanguageChangedEventArgs.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This class holds information about the CurrentLanguageChanged event.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class CurrentLanguageChangedEventArgs
+    {
+        internal CurrentLanguageChangedEventArgs(string previous, string current)
+        {
+            PreviousLanguage = previous;
+            CurrentLanguage = current;
+        }
+
+        /// <summary>
+        /// The previous language.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string PreviousLanguage
+        {
+            get;
+            internal set;
```

Comment:
Change to private
```suggestion
            private set;
```

No need setter if you only set on constructor
```suggestion

```